htdocs/templates2/ocstyle/search.result.caches.tpl

```diff
@@ -48,6 +48,10 @@
     </div>
 {/if}
 
+{if $invalid_waypoints}
+    <p class="errormsg">{t}The following waypoints are invalid and could not be added to the list:{/t}&nbsp; {$invalid_waypoints}</p>
```

Comment:
XSS Attack possible. Please make sure to escape variables which come through a request.

Example Exploit: http://local.opencaching.de/search.php?searchto=searchbylist&listid=1&showresult=1&f_disabled=0&f_inactive=0&f_ignored=1&sort=byname&invalidwp=%3Cscript%20type=%22text/javascript%22%3Ealert(%27Hack%27);%3C/script%3E